hrjobcontract/CRM/Hrjobcontract/Import/EntityHandler/HRJobLeave.php

```diff
@@ -22,33 +33,4 @@ public function handle(array $params, CRM_Hrjobcontract_DAO_HRJobContractRevisio
       'jobcontract_revision_id' => $contractRevision->id,
     ])['values'][0];
   }
-
```

Comment:
No reason really. Just thought the handle function can accommodate the logic without the need for an extra function